modules/operator-georepl-site-removal.adoc

```diff
@@ -0,0 +1,63 @@
+:_content-type: PROCEDURE
+[id="operator-georepl-site-removal"]
+= Removing a geo-replicated site from your {productname} Operator deployment
+
+By using the following procedure, {productname} administrators can remove sites in a geo-replicated setup. 
+
+.Prerequisites 
+
+* You are logged into {ocp}.
+* You have configured {productname} geo-replication with at least two sites, for example, `usstorage` and `eustorage`. 
+* Each site has its own Organization, Repository, and image tags. 
+
+.Procedure 
+
+. Enter the following command to identify your `Quay` database pods:
++
+[source,terminal]
+----
+$ oc get pod -l quay-component=postgres -n  <quay-namespace> -o jsonpath='{.items[0].metadata.name}'
+----
++
+.Example output
++
+[source,terminal]
+----
+quay390useast2-quay-app-5779ddc886-2drh2
+quay390eustorage2-quay-app-66969cd859-n2ssm
+----
+
+. Enter the following command to open an interactive shell session in the `useast2` pod:
++
+[source,terminal]
+----
+$ oc rsh quay390useast2-quay-app-5779ddc886-2drh2
+----
+
+. Enter the following command to permanently remove the `eustorage2` site:
++
+[IMPORTANT]
+====
+The following action cannot be undone. Use with caution.
+====
++
+[source,terminal]
+----
+sh-4.4$ python -m util.removelocation euwest2
+----
++
+.Example output
++
+[source,terminal]
+----
+/app/lib/python3.9/site-packages/tzlocal/unix.py:141: SyntaxWarning: "is not" with a literal. Did you mean "!="?
```

Comment:
These is the issue of https://issues.redhat.com/browse/PROJQUAY-5650, and will be fixed in Quay 3.9.0

Reply:
Removed. Thanks.